tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py36
+envlist = py37
```

Comment:
Change it back to py36. You can use a few different ways of overriding to fit your Python version.

https://tox.readthedocs.io/en/latest/config.html#conf-envlist